packages/simple-hub/env.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+// FIXME: To remove @statechannels/hub's dependency on @statechannels/devtools,
```

Comment:
This FIXME is only here to avoid needing this dependency for `hub` on Heroku. You probably should just use `@statechannels/devtools` for now on this package? Unless you want to get this on Heroku right now to test it against the `xstate-wallet` `rps`.

Reply:
Got it. It's likely that the `simple-hub` will be on Heroku shortly.